src/Location/Location.tsx

```diff
@@ -4,10 +4,7 @@ import './Location.scss'
 const Location: React.FC = () => {
 	return (
 		<section className="location-data">
-			<p>City, State</p>
-			<p>Country</p>
-			<p>US AQI: </p>
-			<p>🌪</p>
+			<p></p>
```

Comment:
Thanks for removing this! I didn't mean to leave it in. Once I get some data fetched from this new weather API, I will add some p tags back in to hold and display the data. I'd love to get your tips and tricks on styling so it looks consistent with the rest of the app!